### ISSUE TITLE
Ошибка MySQL

### ISSUE THREAD

**Issue by wiyegate** [WendyH/api.lostcut.net#4]
Перестал работать api и Ваш форум.
Получаю сообщения

На форуме:
Unable to connect to MySQL and select database.
MySQL reported: No such file or directory

У api:
SQLSTATE[HY000] [2002] No such file or directory [/var/www/api.lostcut.net/f3/lib/db/sql.php:519]

И так уже 2ой день

**Comment by wiyegate** [comment]
Заметил что http://api.lostcut.net/hdkinoteatr/#use4free на этой странице написано

> Архив создаётся ежедневно, автоматически.

А ниже

> Wed Oct 31 2018

т.е. не сегодняшнее число и не вчерашнее

**Comment by WendyH** [comment]
Да, спасибо, буду смотреть. Ещё не разбирался с проблемой. Разберусь сегодня ночью или завтра.

**Comment by wiyegate** [comment]
Теперь всё работает, и форум, и api и база свежая.
Мне нравится Ваш проект, спасибо за Ваши труды.

**Comment by wiyegate** [comment]
Снова не работает api и форум, получаю ошибку
SQLSTATE[HY000] [2002] No such file or directory [/var/www/api.lostcut.net/f3/lib/db/sql.php:519]

**Comment by wiyegate** [comment]
http://api.lostcut.net/hdkinoteatr/videos
http://api.lostcut.net/hdkinoteatr/videos?min_imdb=6&ord=-year&hr=1

разные ошибки со статусом 500

**Comment by WendyH** [comment]
Кто-то там меня усилинно эксплуатирует. Аж место кончилось для временных файлов.
Вроде починил.
